app/src/main/java/com/fstyle/androidtrainning/screen/main/playingfragments/listsong/ListSongRecyclerAdapter.java

```diff
@@ -0,0 +1,108 @@
+package com.fstyle.androidtrainning.screen.main.playingfragments.listsong;
+
+import android.content.Context;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+import com.fstyle.androidtrainning.R;
+import com.fstyle.androidtrainning.model.Track;
+import com.like.LikeButton;
```

Comment:
ERROR: Unused import: com.like.LikeButton. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports